esp/src/src/store/Memory.ts

```diff
@@ -70,6 +70,9 @@ export class Memory<T extends BaseRow = BaseRow> extends BaseStore<T, T> {
     }
 
     protected fetchData(request: QueryRequest<T>, options: QueryOptions<T> = {}): ThenableResponse<T> {
+        if (!options) {
```

Comment:
Is this for the scenario where fetchData was called and like an explicit nullish value was passed in for options, so the default wasn't used?

Reply:
Not sure - I think I hit it while debugging, so just added the extra check...